components/centraldashboard/tsconfig.json

```diff
@@ -1,25 +1,19 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "esModuleInterop": true,
-        "target": "es6",
-        "noImplicitAny": true,
-        "moduleResolution": "node",
-        "sourceMap": true,
-        "outDir": "dist",
-        "baseUrl": ".",
-        "plugins": [
-            {
-                "name": "typescript-tslint-plugin"
-            }
-        ]
-    },
-    "include": [
-        "app",
-        "types"
-    ],
-    "exclude": [
-        "app/**/*_test.ts",
-        "app/**/test_resources.ts",
+  "compilerOptions": {
```

Comment:
Can we please keep the tabbing at 4 spaces?

Reply:
Sorry, I think my editor is inheriting settings from g3 default which uses 2. I just wanted to fix the trailing comma here which is invalid JSON.

Reply:
I see